controllers/champions.js

```diff
@@ -0,0 +1,97 @@
+const models = require('../models')
+
+const getAllChampions = async (request, response) => {
+  try {
+    const allChampions = await models.champions.findAll({
+      include: [{
+        model: models.regions,
+      },
+      {
+        model: models.roles,
+      }],
+    })
+
+    return response.send(allChampions)
+  } catch (error) {
+    return response.status(500).send('Could not find champions, please try again.')
+  }
+}
+const getChampionsByIdWithRegionAndRoles = async (request, response) => {
+  try {
+    const { id } = request.params
+
+    const matchingChampions = await models.champions.findOne({
+      include: [{
+        model: models.regions,
+      },
+      {
+        model: models.roles,
+      }],
+      where: {
+        [models.Op.or]: [
+          { id },
+          { name: { [models.Op.like]: `%${id.toLowerCase()}%` } }
+        ]
+      }
+    })
+
+    return matchingChampions
+      ? response.send(matchingChampions)
+      : response.sendStatus(404)
+  } catch (error) {
+    return response.status(500).send('Could not retrieve the champion, please try again.')
+  }
+}
```

Comment:
This is getting by id OR name. You should have that documented and fix up your naming to match that fact